pom.xml

```diff
@@ -4200,4 +4200,5 @@
       <url>file:///tmp</url>
     </site>
   </distributionManagement>
+  <!-- tickle the build -->
```

Comment:
Why is this one necessary?

Reply:
It's only necessary to get the build bot to run the shadedjars test. Notice that it's in a separate commit, one that I will not merge.